static/main.js

```diff
@@ -10,6 +10,7 @@ const TIME_FORMAT = 'YYYY-MM-DD'
 const COLOR_CONFIRMED = 'rgb(244,67,54)'
 const COLOR_RECOVERED = 'rgb(25,118,210)'
 const COLOR_DECEASED = 'rgb(55,71,79)'
+const COLOR_INCREASE = 'rgb(123,71,19)'
```

Comment:
```suggestion
const COLOR_INCREASE = 'rgb(163,172,191)'
```
Updated color to grey